Automattic/woocommerce-payments
# Fix some i18n issues in checkout

Fixes #5262 

#### Changes proposed in this Pull Request

This fixes a few non localized strings. The full list will hopefully be fixed by this another PR #6385 









#### Testing instructions




Manual testing
* On `develop` run `npm run build`
* Navigate to `languages/woocommerce-payments.pot` and search for `This payment method can not be saved for future use.`. This will return 0 results.
* Now with this PR, re-run `npm run build`
* Go to `languages/woocommerce-payments.pot` and search for `This payment method can not be saved for future use.`. This will return 1 result.
* Search for `An error was encountered`, this should give you the second string we are fixing here.

-------------------


**Post merge**
